source/partials/public_api/api_usage/office/index.md.erb

```diff
@@ -332,4 +336,7 @@ Attribute | Type |
 <span class="response-data-remarks">This is the Unix timestamp representing the date that this <span class='entity-name'>Office</span> was created in the MoxiWorks system.</span>
 
 <span class="response-data-head">deactivated_timestamp</span>
-<span class="response-data-remarks">This is the Unix timestamp representing the date that this <span class='entity-name'>Office</span> was deactivated in the MoxiWorks system. This will be returned null if the office is still active.</span>
\ No newline at end of file
+<span class="response-data-remarks">This is the Unix timestamp representing the date that this <span class='entity-name'>Office</span> was deactivated in the MoxiWorks system. This will be returned null if the office is still active.</span>
+
+<span class="response-data-head">company_programs</span>
+<span class="response-data-remarks">A list of the company specific program names in which the agent participates or is a member.</span>
```

Comment:
Because this is on the office entity, should this say (same below):
`...agent's office...`